compiler/noirc_frontend/src/monomorphization/debug.rs

```diff
@@ -134,11 +135,15 @@ impl<'interner> Monomorphizer<'interner> {
             .clone();
         let mut cursor_type = &var_type;
         for i in 0..arity {
-            if let Some(HirExpression::Literal(HirLiteral::Integer(fe_i, i_neg))) =
+            if let Some(HirExpression::Literal(HirLiteral::Integer(value_i))) =
                 hir_arguments.get(DEBUG_MEMBER_FIELD_INDEX_ARG_SLOT + i)
             {
+                let fe_i = crate::utils::big_int_to_field_element(value_i);
+                let i_neg = value_i.is_negative();
+
+                // TODO: check if there's a way to turn BigInt into i128 without going through FieldElement
```

Comment:
Let me know if there's a way to do this in another way. If not, we can remove this TODO.